src/parsers/forecast_parser.jl

```diff
@@ -254,7 +254,7 @@ function make_forecast_array(all_components, parsed_forecasts::Vector{Dict})
 function write_to_json(filename,Forecasts_dict)
     for (type_key,type_fc) in Forecasts_dict
         for (device_key,device_dicts) in type_fc
-            stringdata =JSON.json(device_dicts, 3)
+            stringdata =JSON2.json(device_dicts, 3)
```

Comment:
Does this work?  I don't think JSON2 has the same functions.

Reply:
It passed all the tests

Reply:
I don't believe you.  :)

Reply:
Perhaps those functions don't get called.

Reply:
Probably not, we would be able to see that in the Codecov report